### PR TITLE
Silicon change

### PR DESCRIPTION
**Pull request by Dream-Master**
* https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/287
https://drive.google.com/file/d/17QikbgicnhJRQxdr3eW62P3mYV-78qNT/view?usp=sharing

### CODE REVIEW

**Comment by GlodBlock** [comment]
why silicon electronic grade only obtainable from crush or cutting silicon boule? it is impossible to have monocrystal silicon **dust** or smelt monocrystal silicon dust into monocrystal silicon ingot. crystal structure will be disrupted when you crush or heat it

**Comment by GlodBlock** [comment]
> why silicon electronic grade only obtainable from crush or cutting silicon boule? it is impossible to have monocrystal silicon **dust** or smelt monocrystal silicon dust into monocrystal silicon ingot. crystal structure will be disrupted when you crush or heat it. they are more similar to gem material instead of metal

**Comment by Dream-Master** [comment]
> why silicon electronic grade only obtainable from crush or cutting silicon boule? it is impossible to have monocrystal silicon **dust** or smelt monocrystal silicon dust into monocrystal silicon ingot. crystal structure will be disrupted when you crush or heat it. they are more similar to gem material instead of metal

Mh any suggestion how we can change this? or should we remove SIeg completly so you can only make boules with Sisg ?

**Comment by GlodBlock** [comment]
i have a rough idea to get SIeg:
silicon boule -- crush --> SIeg crystal nucleus
SIeg crystal nucleus + molten SIsg -- fluid solidifer --> SIeg ingot

**Comment by Pxx500** [comment]
Simple solution is to remove "(Mono Si)" from items name since it doesn't make sense, but having Sieg doesn't make any sense in itself since it's made just like Sisg (getting it from cutting is still counts) so they're technically the same thing.

**Comment by Dream-Master** [comment]
> Simple solution is to remove "(Mono Si)" from items name since it doesn't make sense, but having Sieg doesn't make any sense in itself since it's made just like Sisg (getting it from cutting still counts) so they're technically the same thing.

Ok will look tomorrow into it.

**Comment by GlodBlock** [comment]
they are not the same thing, after the poly silicon become silicon boule, they are mono silicon. if you still don't understand i can explain that in chinese @Pxx500

**Comment by Pxx500** [comment]
I understand that they are different things but when monocrystaline silicon is broken into a powder it's properties are effectively lost and it can be considered polycrystaline (not because it's polycrystaline but because it's useless as a monocrystaline silicone [unless you add a process to use it as a crystal base for more boules])

**Comment by GlodBlock** [comment]
> I understand that they are different things but when monocrystaline silicon is broken into a powder it's properties are effectively lost and it can be considered polycrystaline (not because it's polycrystaline but because it's useless as a monocrystaline silicone [unless you add a process to use it as a crystal base for more boules])

https://github.com/GTNewHorizons/GT5-Unofficial/pull/953#issuecomment-1107798681

**Comment by Pxx500** [comment]
"they are the same thing" as in their properties are essentially the same since monocrystaline powder is similar to polycrystaline

**Comment by leagris** [comment]
I think this detail from reality, does not matter at all in the scope of GTNH. `Silicon` is good enough for this context.

**Comment by Glease** [comment]
The whole spirit of this change is realism. I don't value this as much as others do but I guess at this point we need to emphasis this a bit in this PR.

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -1241,6 +1241,37 @@ public void run() {
...
+        GT_Values.RA.addChemicalRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Quartzite, 2), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Silicon, 1), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesia, 2), 100, 8);
+        GT_Values.RA.addChemicalRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.CertusQuartz, 1), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Silicon, 1), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesia, 2), 100, 8);
+        GT_Values.RA.addMultiblockChemicalRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(1), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 4), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sulfur, 4)}, null, null, new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Silicon, 3), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 2)}, 10, 120);
```
Comment (GlodBlock) [inline src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:1249]: `GT_Values.RA.addMultiblockChemicalRecipe(new ItemStack[]{GT_Utility.getIntegratedCircuit(1), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 9), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 4)}, null, null, new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Silicon, 3), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10)}, 10, 120)`

**Review by GlodBlock — CHANGES_REQUESTED** [review]
strange chemical recipe

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:1249]: will check why that was added/changed

**Pull request merged**